src/styles/_components/_modals.scss

```diff
@@ -37,7 +37,8 @@
         right: 24px;
     }
 
-    &__close-button--light svg {
+    &__close-button--light svg,
+    &__close-button--light .st0 {
```

Comment:
why do you need to attach the styling this with the svg class? can you attach it with the tag instead?